keps/sig-api-machinery/2161-apiserver-default-labels/README.md

```diff
@@ -39,10 +39,10 @@
 Items marked with (R) are required *prior to targeting to a milestone / release*.
 
 - [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
-- [ ] (R) KEP approvers have approved the KEP status as `implementable`
-- [ ] (R) Design details are appropriately documented
-- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
-- [ ] (R) Graduation criteria is in place
+- [x] (R) KEP approvers have approved the KEP status as `implementable`
+- [x] (R) Design details are appropriately documented
+- [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [x] (R) Graduation criteria is in place
```

Comment:
You need PRR approval like this: https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-api-machinery/1040.yaml

I recommend @deads2k to be the approver.

Reply:
done thx

Reply:
wait,  i actually was missing the new file, i see, this is part of the new prod-readiness wg stuff... ok, added the file, thanks for the pointer 

done (again :))